superset-frontend/src/visualizations/FilterBox/FilterBoxChartPlugin.js

```diff
@@ -22,9 +22,12 @@ import thumbnail from './images/thumbnail.png';
 import controlPanel from './controlPanel';
 
 const metadata = new ChartMetadata({
+  category: t('Tools'),
   name: t('Filter box'),
-  description:
-    'A multi filter, multi-choice filter box to make dashboards interactive',
+  description: t(`Chart component that lets you add a custom filter UI in your dashboard. When added to dashboard, a filter box lets users specify specific values or ranges to filter charts by. The charts that each filter box is applied to can be fine tuned as well in the dashboard view.
+
+    Note that this plugin is being replaced with the new Filters feature that lives in the dashboard view itself. It's easier to use and has more capabilities!`),
+  tags: [t('Predictive')],
```

Comment:
Filterbox is not predictive... something went funky here.

Reply:
Oh you're right

Reply:
```suggestion
```